src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 100%;
+  flex-grow: 1;
+}
+
+.b1 {
+  background-color: rgb(255,0,0);
+  order: 1;
+}
+
+.b2 {
+  background-color: rgb(204,0,0);
+  order: 2;
+}
+
+.b3 {
+  background-color: rgb(153,0,0);
+  order: 3;
+}
+
+.b4 {
+  background-color: rgb(102,0,0);
+  order: 4;
+}
+
+.b5 {
+  background-color: rgb(51,0,0);
+  order: 5;
+}
+
+.b6 {
+  background-color: rgb(0,0,0);
+  order: 6;
+}
+
+@media (min-width: 600px) {
+  .block {
+    flex-basis: 50%;
+  }
+
+  .b3 {
+    order: 4;
+  }
+
+  .b4 {
+    order: 3;
+  }
+
+  .b5 {
+    order: 5;
+  }
+
+}
+
+@media (min-width: 900px) {
+  .block {
+    flex-basis: 33%;
+  }
```

Comment:
33 + 33 + 33 is 99, not 100
that's why when this media query is applied you get this
![Screenshot 2022-03-21 at 22 31 41](https://user-images.githubusercontent.com/79282749/159358796-5a39b143-8fe9-440b-a28b-399d3a816198.jpg)

the white line on the right, because it does not cover 100%. only 99